RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -10,14 +10,20 @@
     import RxSwift
 #endif
 
```

Comment:
I think we should document this, what `completed` means, what `failed` means and what is this `enum` for.